### DISCUSSION TITLE
RoboticsAcademy is now on goodfirstissue.dev

### DISCUSSION THREAD

**Discussion by akshaynarisetti** [JdeRobot/RoboticsAcademy#1720] [Announcements]
Hello Everyone, 👋 

RoboticsAcademy is now a part of https://goodfirstissue.dev/

Thank you @pariaspe for the contribution.

<img width="1316" alt="Screenshot 2022-05-16 at 8 14 32 PM" src="https://user-images.githubusercontent.com/58532023/168621704-01536b24-a040-4b66-87a7-69572841d473.png">

**Comment by jmplaza** [comment]
Well done!
